src/server/auth/server/testing/auth_test.go

```diff
@@ -2498,6 +2505,10 @@ func TestDebug(t *testing.T) {
 	tu.DeleteAll(t)
 	defer tu.DeleteAll(t)
 
+	// Activate auth at the beginning of the test, so pipelines are created with auth tokens.
+	// Otherwise when we activate auth the pps master will asynchronously update RCs with tokens,
+	// which can cause pods to stop existing while we collect the debug logs.
+	adminClient := tu.GetAuthenticatedPachClient(t, auth.RootUser)
```

Comment:
/cc @brycemcanally - you mentioned the Debug dump test was flaky with auth enabled - this explains why the containers were disappearing during the test and fixes it.

Reply:
I'm not following why this addresses the issue. Does the `tu.GetAuthenticatedPachClient` call below when setting up the alice client not activate auth?

Reply:
Yeah I realized my comment is not quite accurate. Every call to `GetAuthenticatedPachClient` calls `pps.ActivateAuth`, which rotates the auth tokens for any existing pipelines, even if auth is already enabled. So setting up the alice client enables auth, but getting the root token attempts to activate auth again, which causes the specs for the existing pipelines to changge and bounces the pods.